web/app/scripts/views/map/layers-controller.js

```diff
@@ -499,25 +524,32 @@
          * blackspot layer and adds a new one to the layer group with
          * the updated URL
          */
-        function updateBlackspotLayer(blackspotsUrl, blackspotsUtfGridUrl) {
+        function updateBlackspotLayer(blackspotsUrl, blackspotsUtfGridUrl, tilekey) {
+            // TODO: update to pass in a tilekey which will filter on polygon
```

Comment:
Is this TODO still applicable? If there is some missing polygon filtering functionality, I think it should be cleared up before merging.